packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksEditor.tsx

```diff
@@ -161,6 +162,7 @@ const pipe =
     fns.reduce<Editor>((prev, fn) => fn(prev), value);
 
 interface BlocksEditorProps {
```

Comment:
I think you need to start building compound types e.g. `placeholder` & `ariaLabel` are passed to `BlocksContent` so why not use that prop type to build this one? It'll be easier to edit those types in the future imo.

Reply:
sure 👍